server/src/test/java/org/elasticsearch/index/IndexingSlowLogTests.java

```diff
@@ -500,7 +500,7 @@ public void testSetLevels() {
     }
 
     private void assertTimeValueException(final IllegalArgumentException e, final String key) {
-        final String expected = "illegal value can't update [" + key + "] from [-1] to [NOT A TIME VALUE]";
+        final String expected = "illegal value can't update [" + key + "] from [-1nanos] to [NOT A TIME VALUE]";
```

Comment:
Likewise here, let's avoid leaking any minus-one-with-units cases to users.

It's probably a bug that the slowlog thresholds default to `-1ns`. Can we just update them to be `TimeValue.MINUS_ONE` in the production code?